### PR TITLE
Add Azure DevOps integration details to CI section

### PR DESCRIPTION
**Pull request by kimsauce**
### Description

Added Azure page back to Sidebars and replaced plugin information with new guidance.

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Documentation fix (typos, incorrect content, missing content, etc.)

### Checklist

- [ ] I have read the [contributing](https://github.com/saucelabs/sauce-docs/blob/main/docs/contributing.md) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by nadvolod** [comment]
@kimsauce which problem are we solving here by deleting this file?

**Review by nadvolod — APPROVED** [review]

File: docs/ci/azure.md [hunk 1]
```diff
@@ -1,9 +1,233 @@
...
+  ```js
+  x
+  ```
+
+  </TabItem>
+  <TabItem value="Java">
+
+  ```java
+  x
+  ```
```
[the remark this answers is not in the document]
Comment (kimsauce) [reply docs/ci/azure.md:52]: @mdsauce @nadvolod , just following up on this

**Review by nadvolod — APPROVED** [review]

File: docs/ci/azure.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (kimsauce) [reply docs/ci/azure.md:52]: Hi @nadvolod , I saw that you'd marked this PR approved, but it still needs this missing information. Or I could remove these code snippets.

**Review by lysannep — APPROVED** [review]
LGTM! (-missing code snippets)

File: docs/ci/azure.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (kimsauce) [reply docs/ci/azure.md]: Hi @nadvolod , I've removed this section and left the C# example there. The changes will be live in about 10 mins (https://docs.saucelabs.com/ci/azure/).

**Pull request merged**